### ISSUE TITLE
[BUG] - Access blocked moosync.app has not completed Googl's verification process

### ISSUE THREAD

**Issue by L4znet** [Moosync/Moosync#243]
HI everyone!
Trying to connect my YouTube account, complete all the process, but I receive this error (look at the screen)
![image](https://github.com/Moosync/Moosync/assets/101733611/301cbc5c-a0f9-48c8-b4e7-98e934fa1000)

I don't really how to reproduce this bug, but I've been tried to recreate multiple time my app, it's doesn't work.

Have a good day!

**Comment by sree-vignesh** [comment]
How is the issue now ? Has it been fixed ?

**Comment by Ovenoboyo** [comment]
Hi, what version of the app are you guys on?

**Comment by L4znet** [comment]
Hi!
I'm on the 8.1.1, I downloaded the app the same day as I created the issue

**Comment by Ovenoboyo** [comment]
Did you create your own credentials on Google cloud platform?

**Comment by L4znet** [comment]
Give me a minute, I'll check that

**Comment by L4znet** [comment]
> How is the issue now ? Has it been fixed ?

Nope

**Comment by AkiyamaMizuki3** [comment]
I had a similar issue before. Try adding a your's (developer's) email to the testers list

**Comment by AryRoy** [comment]
You need to publish (or push) in your oauth consent page and then it will give you a warning, the page will then show you this instead of your previous error. Just click on the "advanced" text and press "Proceed to link anyway" and you should be done tbhere

![image](https://github.com/Moosync/Moosync/assets/118198048/518933d1-a14c-4607-a026-7668630dc625)

**Comment by Ovenoboyo** [comment]
Even without publishiny your oauth consent screen, you should be able to access the right page if you add your email to the testers email list as @AkiyamaMizuki3 suggested.

**Comment by Kamek437** [comment]
Still doing this to me, and I've looked everywhere for this gcloud project testers list but seriously can't find it. I'd assume that the owner role is as good as the tester role but...dunno.

**Comment by Kompreya** [comment]
Adding to this - I just installed v10.3.2 and followed the wiki documentation. After entering the key and secret and attempting to login via Moosync to YouTube, I was directed to login via browser and received an error 403. I also ensured that the email I used to set up the API was the same as what I use to log into YouTube with.

@AkiyamaMizuki3 has the correct solution. Identify the email of the youtube account you want to use for logging into Moosync, and add that email to the test users section of the API page you set up via wiki instructions, like so:

![image](https://github.com/Moosync/Moosync/assets/41587427/845bdd46-259c-41d4-9a14-7ac93c54936d)

I'm not sure if this is the "correct" way to go about this, so perhaps the wiki needs to be updated.
